app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3:
+        print("Enter the correct command.")
+        return
+    _, file_copy, file_paste = command
```

Comment:
Also validate that command is really cp
Because as for me if the command will be:
"hello file.txt file2.txt"
Is we need a copy file in this case?